deeplake/enterprise/dataloader.py

```diff
@@ -102,7 +102,7 @@ def __init__(
     def __len__(self):
         round_fn = math.floor if self._drop_last else math.ceil
         return round_fn(
-            len(self.dataset) / ((self._batch_size or 1) * self._world_size)
+            len(self._orig_dataset) / ((self._batch_size or 1) * self._world_size)
```

Comment:
I think @istranic was mentioning that we had problems with denominator, nut numerator

Reply:
The return of this method is the denominator.